### ISSUE TITLE
aria-labelledby references for the group element that represents our MenuButton are not getting announced by JAWS.

### ISSUE THREAD

**Issue by havvamehfooz** [FreedomScientific/VFO-standards-support#949]
You can check with the following native HTML sample:
https://jsfiddle.net/e0actL9r/2/

There are five small regions in this sample:
1) table region
2) group region (represents our sap.m. MenuButton -> visually indicated by two small buttons without text)
3) grid region
4) toolbar  region (represents our sap.m. MenuButton within a toolbar-> visually indicated by two small buttons without text)
5) table region

All of those regions have a single focusable element so we can pass through them.

The issue is that the aria-labelledby references for the group element that represents our sap.m.MenuButton are not getting announced by JAWS.

Steps to reproduce: 
1) Tab from the table down to the first “group” element (the two small empty buttons)
**Observe:**
The aria-labelledby references are not announced. We would hear only the aria-keyshortcuts announcement and that the element is a “group”

2) Tab to the next “group” element that is nested within a toolbar ( number 5 that represents a sap.m.MenuButton with a two small empty buttons visually)
**Observe:**
We receive full announcements including the aria-labelledby and the aria-keyshortcuts texts.

The case from the application is related to lacking announcements for aria-labelledby, which in the sample is the “group” element (number 2) without a toolbar wrapper.

The same issue is not reproducible with NVDA.

Seems that JWAS tries to switch modes between the different regions (From table to group and from grid to group) unsuccessfully or something.
If we simply have interactive elements like buttons and the group element is in between the issue won’t be reproducible like in this sample:
https://jsfiddle.net/s9y82uog/
